sort_all_json.py

```diff
@@ -6,6 +6,10 @@
         if 'task_chain' in str(fname):
             continue
         with open(fname) as f:
-            js = json.load(f)
+            try:
```

Comment:
Note: unrelated changes (Running this also made it hard to verify whether `tier1.json` had been sorted or sorted and had entries added)

Reply:
sorry, i'll segment my PRs next time